### DISCUSSION TITLE
What is the best approach to moving resources to another subscription?

### DISCUSSION THREAD

**Discussion by vinicius-batista** [pulumi/pulumi-azure-native#2357] [Q&A]
Currently, all of our Azure resources are deployed in a single subscription, with multiple stacks that have interdependencies through their use of outputs. However, we've found it challenging to manage these resources effectively, and are planning to transition to an approach that leverages Azure Landing Zones with multiple subscriptions.

To achieve this, we're exploring options to move our Pulumi-managed resources to a different subscription. While Azure offers its own resource mover, we've encountered issues with updating the subscriptionId configuration field: the refresh and up commands don't recognize the changes.

We're looking for guidance or documentation to help us make the move to a new subscription with minimal downtime. Can you recommend any resources or best practices to help us achieve this?

**Comment by wesleyteixeira** [comment]
I’m facing a similar challenge but in the reverse direction. Currently, I have multiple Azure subscriptions and want to consolidate everything into a single subscription separated by resource groups.
